MiniApp/Classes/core/MiniAppAnalytics.swift

```diff
@@ -49,7 +49,6 @@ public class MiniAppAnalyticsLoader: NSObject {
 
 public class MiniAppAnalytics {
     public static let notificationName = Notification.Name("com.rakuten.esd.sdk.events.custom")
-    internal static let acc = "1553", aid = "1"
```

Comment:
You have to find a way to still keep this ids when  `sendAnalytics` is call with no `analyticsConfig` parameter. I suggest to have a static `MAAnalyticsConfig(acc: "1553", aid: "1")` you can put in the fallback array when `analyticsConfig` is nil

Reply:
Cool. That's a good idea. Added them now